x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -35,6 +36,10 @@ const (
 	maxRequestRetries = 30              // maximum number of retries for HTTP requests
 )
 
+var (
+	tokenNameStrip = regexp.MustCompile(`\s\([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\)$`)
```

Comment:
can you add a comment here why and what this is?

Reply:
added